test/integration/generated/datatables/README.md

```diff
@@ -0,0 +1,27 @@
+## Azure GeneratedClient SDK for JavaScript
+
+This package contains an isomorphic SDK for GeneratedClient.
+
+### Currently supported environments
+
+- Node.js version 8.x.x or higher
+- Browser JavaScript
+
+### How to Install
+
+```bash
+npm install datatables
+```
+
+### How to use
+
+#### Sample code
+
+Refer the sample code in the [azure-sdk-for-js/samples](https://github.com/Azure/azure-sdk-for-js/tree/master/samples) folder.
+
+## Related projects
+
+- [Microsoft Azure SDK for Javascript](https://github.com/Azure/azure-sdk-for-js)
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2Fsdk%2Fcdn%2Farm-cdn%2FREADME.png)
```

Comment:
the path here is for `sdk/cdn/arm-cdn/README` - if we can generate this to match the package name, that'd be good